index.html

```diff
@@ -1760,6 +1760,11 @@ <h3>
                   Batch Operations
               </h3>
 
+              <p class="issue atrisk" title="Implementation of Batch Endpoint">
+                  The Secure Data Storage Working Group is seeking implementer feedback regarding
+                  this feature. See also <a href="https://github.com/decentralized-identity/edv-spec/issues/4"> Issue #4</a>.
```

Comment:
```suggestion
There is an ongoing debate in the Secure Data Storage Working Group regarding whether or not a batch API is necessary to achieve acceptable levels of performance. The Secure Data Storage Working Group is seeking implementer feedback regarding this feature, or alternatively, the use of parallel requests to achieve similar performance characteristics. See also <a href="https://github.com/decentralized-identity/edv-spec/issues/4"> Issue #4</a>.
```

Reply:
Looks good - I've updated the PR to use your suggested text.